lib/data/game.js

```diff
@@ -3,5 +3,6 @@ GridGame.data.game = {
   height: 12,
   turn_time: 100,
   spawn_interval: 3,
-  city_conversion_size: 10
+  city_conversion_size: 10,
+  rock_count: 100
```

Comment:
Small tip: when I’m writing a collection like this, where each entry is on a new line, and there’s a comma at the end of the previous entry, I add a trailing comma, like so:

```javascript
  city_conversion_size: 10,
  rock_count: 100,
};
```

Most languages will ignore the trailing comma, but it makes the diffs slightly cleaner if I ever need to extend the collection.